### DISCUSSION TITLE
Strange folder at SourceForge

### DISCUSSION THREAD

**Discussion by fredericDelaporte** [nhibernate/nhibernate-core#3226] [General] [answered]
At the root of NHibernate files on Sourceforge, there is 5.4.0 folder with a tar.gz archive, which is not what we usually publish.
https://sourceforge.net/projects/nhibernate/files/5.4.0/

The normal 5.4.0 folder is a sub-folder of a NHibernate folder
https://sourceforge.net/projects/nhibernate/files/NHibernate/5.4.0/

What is that unusual 5.4.0 folder at the root?

**Comment by bahusoid** [comment, marked as answer]
Looks like content generated by github release https://github.com/nhibernate/nhibernate-core/releases/tag/5.4.0

**Reply by fredericDelaporte** [reply]
Now there is a 5.3.15 folder too. It seems SourceForge's GitHub integration has been enabled, I do not know by who. But we can see in NHibernate GitHub repo webhooks its setup: https://github.com/nhibernate/nhibernate-core/settings/hooks/376973958

I am not convinced of the value of this. The contained archive files are just source code. I think that integration should be removed, and these folders deleted.

**Reply by oskarb** [reply]
I just know I did not do it. :)

**Reply by fredericDelaporte** [reply]
It may be just a test to see if it has some value, and I am fine with that. But since the result does not seem useful to me, I think we should remove it.

**Reply by oskarb** [reply]
It does look confusing to have both.

**Reply by bahusoid** [reply]
No idea who or why did it. But yeah - let's just remove it.

**Reply by oskarb** [reply]
I see some audit logs that suggest it was @hazzik

**Reply by hazzik** [reply]
yes, it was me, but I then removed it afair.

**Reply by fredericDelaporte** [reply]
I have unchecked the `Active` box of the webhook and cleaned-up the folders on SourceForge.

**Comment by oskarb** [comment]
One could also consider it unfortunate that we publish a source code package at SourceForge and a different source code package at github. The latter is autogenerated I guess... Can we turn that off? Or make it be all the packages that we have traditionally uploaded to sourceforge instead of something that github generates?

**Reply by fredericDelaporte** [reply]
It does not look like we can prevent that GitHub behavior. We can add additional files, so we could add the same files than those we upload on SourceForge. But GitHub will also add its own auto-generated archives of source files. I do not see an option for removing them or turning that off on a draft release.
